src/Sentry/Internal/BackgroundWorkerBase.cs

```diff
@@ -144,7 +125,7 @@ private async ValueTask WorkerAsync()
                         }
                         catch (Exception exception)
                         {
-                            _options.DiagnosticLogger?.LogError(
+                            Options.DiagnosticLogger?.LogError(
```

Comment:
Seems the format on this one expects 3 arguments but if the first is Exception (not part of `params`) are we missing one?

Reply:
Can't figure out what it's missing :/
Probably some info got lost in transition from events to envelopes. I'll remove the extra placeholder for now.